src/ResourceManager/ServiceBus/Commands.ServiceBus/packages.config

```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.ServiceBus" version="1.0.0" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.ServiceBus" version="1.0.3" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
```

Comment:
Remove these three new dependencies - they are imported by the common dependencies and thus not needed here

Reply:
removed